package.json

```diff
@@ -53,7 +53,8 @@
     "husky": "^8.0.1",
     "jest": "^29.0.0",
     "netlify-plugin-cypress": "^2.2.1",
-    "prettier": "^2.0.0"
+    "prettier": "^2.0.0",
+    "strip-ansi": "^7.0.1"
```

Comment:
Added to be able to more clearly compare log lines containing chalk colouring without needing to add the weird escape characters in our fixtures